### PR TITLE
Add files.

### PR DESCRIPTION
**Pull request by jbej**

### CODE REVIEW

File: src/components/TodoList.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import style from './TodoList.css';
+import Todo from './Todo';
+import TodoForm from './TodoForm';
+
+
```
Comment (matwjablonski) [inline src/components/TodoList.js]: why to many empty lines?

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './containers/App';
+import{ AppContainer } from 'react-hot-loader';
+
+
+
+if (module.hot) {
+	module.hot.accept('./containers/App', () => {
+		const NaxtApp = require('./containers/App').default;
```
Comment (matwjablonski) [inline src/index.js:10]: Naxt? :)

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (jbej) [reply src/index.js:10]: NextApp - because it's in Kodilla's materials.

File: src/components/TodoList.js [hunk 3]
```diff
@@ -3,8 +3,6 @@ import style from './TodoList.css';
 import Todo from './Todo';
 import TodoForm from './TodoForm';
 
-
-
 const TodoList = props => {
 	const todoListItems = props.data.map(item => <Todo key={item.id} id={item.id} todo={item.text}
 		remove={props.remove} iform={props.iform}/>)
```
Comment (matwjablonski) [inline src/components/TodoList.js]: add semicolon

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**